modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageTable.es.js

```diff
@@ -25,7 +26,10 @@ const Table = ({items, totalCount}) => {
 					</ClayTable.Cell>
 
 					<ClayTable.Cell headingCell style={{width: '14%'}}>
-						{Liferay.Language.get('due-date')}
+						<ListHeadItem
```

Comment:
Hey @jotascript, rename it to `dueDate` instead and apply it to other places, if any. Thx Bro